crates/fluvio-spu/src/mirroring/remote/controller.rs

```diff
@@ -492,8 +518,8 @@ where
 
 fn create_backoff() -> ExponentialBackoff {
     ExponentialBackoffBuilder::default()
-        .min(Duration::from_secs(1))
-        .max(Duration::from_secs(300))
+        .min(Duration::from_millis(200))
```

Comment:
this is probably too small.

Reply:
for min or max?

Reply:
Increased min to 1s and max to 30s. Wdyt?